packages/js/src/utils/generators.test.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  CODE_VERIFIER_MAX_LENGTH,
+  generateCodeChallenge,
+  generateCodeVerifier,
+  generateState,
+} from './generators';
+
+describe('generateState', () => {
+  test('with fixed length', () => {
+    const state = generateState();
+    expect(state.length).toEqual(CODE_VERIFIER_MAX_LENGTH);
+  });
```

Comment:
all these `with fixed length` s are not expected

Reply:
Removed.